### DISCUSSION TITLE
Does this library support Federated identity provider (google, fb, etc sign in)

### DISCUSSION THREAD

**Discussion by ancashoria** [gladly-team/next-firebase-auth#608] [General]
I'm trying to find out if I can use this library to login users with Google/Fb/Apple accounts.
I see that firebase can handle this functionality and it's called [Federated identity provider integration.](https://firebase.google.com/docs/auth/).

**Comment by kmjennison** [comment]
Yes, it supports all login methods that Firebase supports.
